lib/fluent/plugin/out_splunk_hec.rb

```diff
@@ -51,6 +53,9 @@ class SplunkHECOutput < ObjectBufferedOutput
 
     def configure(conf)
       super
+
```

Comment:
It should raise an error or warning when users set both `channel` and `auto_generate_channel` because it ignores `channel` in this implementation.